<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/images/images/Database-Encryption.png</normaloff>:/images/images/Database-Encryption.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1361</width>
      <height>681</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Таблица устройств</string>
     </attribute>
     <widget class="QTableView" name="deviceTableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>1341</width>
        <height>641</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>0</y>
        <width>951</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Фильтрация по :</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="FilterComboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <item>
            <property name="text">
             <string>Название устройства</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Тип устройства</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Инвентарный номер</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Местонахождение</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Статус устройства</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLineEdit" name="SearchLine">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="SearchButton">
         <property name="text">
          <string>Поиск</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ClearFilterButton">
         <property name="text">
          <string>Очистить Фильтр</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>План устройств</string>
     </attribute>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>1341</width>
        <height>621</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1366</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="Add_action"/>
    <addaction name="Delete_action"/>
    <addaction name="Find_action"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuRefind">
    <property name="title">
     <string>Refind</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuRefind"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="allowedAreas">
    <set>Qt::LeftToolBarArea|Qt::TopToolBarArea</set>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="Add_action"/>
   <addaction name="Delete_action"/>
   <addaction name="Find_action"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="Add_action">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/add-server-icon.png</normaloff>:/images/images/add-server-icon.png</iconset>
   </property>
   <property name="text">
    <string>Добавить устройство</string>
   </property>
  </action>
  <action name="Delete_action">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/remove-server-icon.png</normaloff>:/images/images/remove-server-icon.png</iconset>
   </property>
   <property name="text">
    <string>Удалить устройство</string>
   </property>
  </action>
  <action name="Find_action">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/search-server-icon.png</normaloff>:/images/images/search-server-icon.png</iconset>
   </property>
   <property name="text">
    <string>Найти устройство</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/exit.png</normaloff>:/images/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
